_data/current.yml

```diff
@@ -188,3 +188,10 @@
   dates: "November 19-22, 2024"
   url: https://agiletestingdays.com/?utm_source=testingconferences
   status: <a href="https://agiletestingdays.com/call-for-papers/?utm_source=testingconferences" target="_blank">CFP is Open</a> until February 11, 2024 | Super Early Bird Pricing is Open until April 26, 2024
+
+  - name: BCS SIGiST Summer Conference 2024 Quantum Leap: the Past, Present and Future of Testing
+  location: Online and in-person in London
+  dates: "June 19, 2024"
```

Comment:
I see this is in June, can you move it to the proper location within in the document? Thanks

Reply:
I hope that this is ok now